packages/ui-default/locales/zh.yaml

```diff
@@ -981,4 +981,7 @@ You've already attended this contest.: 您已经参加本次比赛。
 You've already claimed this homework.: 您已认领过该作业。
 You've already voted.: 您已经投过票。
 Your permission: 您的权限
-Your role: 您的角色
\ No newline at end of file
+Your role: 您的角色
+FUN Contest: 乐多赛制
+single_ratio: 每次错误提交倍率
```

Comment:
Did you tell anyone, in any place, what these options are, and how they affect score calculation?